vignettes/data-transform-as-shiny-module.Rmd

```diff
@@ -12,19 +12,20 @@ vignette: >
 
 ## Introduction
 
-`teal_transform_module()` is a Shiny module that takes `ui` and `server` arguments. When provided, `teal` will execute data transformations for the specified module when it is loaded and whenever the data changes. `server` extend the logic behind data manipulations, where `ui` extends filter panel with new UI elements that orchestrate the transformator inputs.
+`teal_transform_module()` is a Shiny module that takes `ui` and `server` arguments. When provided, `teal` executes data transformations for the specified module both when it is loaded and whenever the data changes.
+The `server` argument extends the logic behind data manipulations, while the `ui` argument adds new UI elements to the filter panel, managing the transformator inputs.
 
-`teal` version `0.16` introduced a new, optional argument in `teal::module` named `transformators`. 
-This argument allows to pass a `list` of `"teal_data_module"` class of objects created using `teal_transform_module()` function.
+In version `0.16`, `teal` introduced a new optional argument in `teal::module` named `transformators`.
+This argument allows users to pass a `list` of objects of `teal_data_module` class, created using the `teal_transform_module()` function.
 
-The main benefit of `teal_transform_module()` is the ability to transform data before passing it 
-to the module. This feature allows to extend the regular behavior of existing modules by specifying custom data operations on data inside this module.
+The main advantage of `teal_transform_module()` is its ability to transform data before passing it to the module.
+This feature allows users to extend the default behavior of existing modules by performing specific data operations on the data within the module.
```

Comment:
Would it be better to reverse these 2 paragraphs? Then the first 2 paragraphs would be introducing `teal_transform_module()`, and then the 3rd talks about how to pass it into `teal::module`

Reply:
Agree.

Reply:
Done in 73c1001c2285b95f123b0cca37a46656d5b7ca12